specs/swagger.yaml

```diff
@@ -198,7 +197,95 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
-
+  /psms:
+     get:
+      summary: Get a collection of peptide spectrum matches  
+      operationId: getPSMs
+      description: Get specific Peptide Spectrum Match (PSM) if is present in the database and has been identified by a previous experiment in the resource. 
+      tags:
+        - spectra
+      produces:
+        - "application/json"
+      parameters:
+        - name: pageSize
+          in: query
+          description: How many items to return at one time (default 100, max 100)
+          required: false
+          type: integer
+          default: 100
+        - name: pageNumber
+          in: query
+          description: Current page to be shown paged datasets (default page 1)
```

Comment:
Remove "paged datasets", since it's not necessary and this end point doesn't return datasets anyway.